### DISCUSSION TITLE
Nix for package management

### DISCUSSION THREAD

**Discussion by tarikozkanli** [koka-lang/koka#567] [General]
Can we use nix as a general package manager for Koka ?

**Comment by TimWhiting** [comment]
On a related note, 

I've also been thinking about Bazel integration, which would be nice as far as bundling and distributing C code in a polyglot language mix. I know Koka currently uses vcpkg for including C libraries, but that seems pretty exclusive to C, and I don't see it scaling to support javascript / web, or koka packages.

Here is the central bazel registry, which has many of the C libraries we would need, but we could create a separate registry specifically for Koka packages, which we could supplement with Koka docs for the libraries.
https://registry.bazel.build/
